systemd/afterburn-checkin.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Descitpion=Afterburn Check In
+After=network-online.target
```

Comment:
This can be dropped.  `network-online.target` is documented to be unreliable, and Afterburn should be automatically retrying requests if the network isn't up yet.

Reply:
:+1: will update